indico/htdocs/js/indico/modules/category_management.js

```diff
@@ -99,5 +110,53 @@
                 error: handleAjaxError
             });
         }
+
+        function updateCategoryDeleteButton() {
+            if ($table.find(categoryRowSelector).length) {
+                $('.banner .js-delete-category').addClass('disabled');
+            } else {
+                $('.banner .js-delete-category').removeClass('disabled')
+                    .attr('title', $T.gettext("Delete category"));
+            }
+        }
+
+        function bulkDeleteButtonTooltipContent() {
+            var $checked = getSelectedRows();
+            if ($checked.length) {
+                if ($bulkDeleteButton.hasClass('disabled')) {
+                    return $T.gettext("At least one selected category cannot be deleted because it is not empty.");
+                } else {
+                    return $T.gettext("Delete {0} selected categories").format($checked.length);
+                }
+            } else {
+                return $T.gettext("Select the categories to delete first.");
+            }
+        }
+
+        function bulkDeleteCategories() {
+            var selectedCategories = getSelectedCategories();
+            ajaxDialog({
+                url: $table.data('bulk-delete-url'),
+                title: $T.gettext("Delete categories"),
+                data: {
+                    category_id: getSelectedCategories()
+                },
+                onClose: function(data) {
+                    if (data && data.success) {
+                        getSelectedRows().remove();
+                    }
+                }
+            });
+        }
+
+        function getSelectedCategories() {
+            return $table.find(categoryRowSelector).find('input[type=checkbox]:checked').map(function() {
+                return $(this).attr('value');
+            }).toArray();
+        }
+
+        function getSelectedRows() {
+            return $table.find(categoryRowSelector).has('input[type=checkbox]:checked');
```

Comment:
ok, fine then - thought jquery would be smart enough to translate those pseudoselectors to proper selectors automatically